src/main/swap_math.yml

```diff
@@ -0,0 +1,24 @@
+  transaction:
+    to: '0xbe7c43a580000000000000000000000000000001'
+    data:
+    - :label computeSwapStep_1 0x100d3f74000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010146dd68287f35899f20af670000000000000000000000000000000000000000000000001bc16d674ec800000000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000258
```

Comment:
Can you add a comment which is the matching unit test?

Reply:
Done